{{cookiecutter.repo_name}}/Makefile

```diff
@@ -0,0 +1,46 @@
+.DEFAULT_GOAL := validate
+
+clean:
+	find . -name '*.pyc' -delete
+	coverage erase
+	rm -rf assets
+
+requirements:
+	pip install -qr requirements/local.txt --exists-action w
+
+validate_python: clean
```

Comment:
I would rather have a command that is `test` or `test_python` that only runs the tests, and a separate `validate` command (like you already have) that runs the tests and the quality.

Right now you have a separate task for quality, but no separate task for just running tests, which doesn't seem optimal. If I'm debugging a ton of tests, I don't care about the quality right just now and don't want to see that output.